### PR TITLE
Logics tests

### PR DESCRIPTION
**Pull request by pink-bunny**

### CODE REVIEW

File: state/movie/operations/__tests__/movie.test.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    it('Dispatch fetchMovieSuccess() with id', () => {
+      expect(fetchMovieSuccess).toHaveBeenCalledWith(id);
+    });
+  });
+
+  describe('failure.', () => {
+    beforeEach(() => {
+      httpClient = mockMultiHttpClient([
+        { method: 'get', response: movieInfoResponseError, reject: true },
+        { method: 'get', response: movieImagesResponseSuccess },
```
Comment (plzen) [inline state/movie/operations/__tests__/movie.test.js]: You don't need to add mocks which aren't used in the test, so it's enough to add only 1 mock response with reject = true

File: state/movie/operations/normalize/movie.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { normalize, schema } from 'normalizr';
+
+const normalizeMovie = (data) => {
```
Comment (plzen) [inline state/movie/operations/normalize/movie.js]: Please, put this file near normalizeMovies function which you have created for other operations

File: state/my_lists/operations/normalize/myLists.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { normalize, schema } from 'normalizr';
+
+export default (data) => {
```
Comment (plzen) [inline state/my_lists/operations/normalize/myLists.js]: Put this file near other normalizers

File: state/session/operations/__tests__/requestSession.test.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      expect(cookie.set).toHaveBeenCalledWith({ sessionId, username });
+    });
+  });
+
+  describe('failure.', () => {
+    beforeAll(() => {
+      httpClient = mockMultiHttpClient([
+        { method: 'get', response: authenticationTokenResponseSuccess },
+        { method: 'post', response: {}, reject: true },
+        { method: 'post', response: authenticationSessionResponseSuccess }
```
Comment (plzen) [inline state/session/operations/__tests__/requestSession.test.js]: You don't need add any mocks after mock which have `reject: true`

File: state/movie/operations/__tests__/movie.test.js (hunk 1 above)
Comment (pink-bunny) [reply state/movie/operations/__tests__/movie.test.js]: Done

File: state/movie/operations/normalize/movie.js (hunk 2 above)
Comment (pink-bunny) [reply state/movie/operations/normalize/movie.js]: Done

File: state/my_lists/operations/normalize/myLists.js (hunk 3 above)
Comment (pink-bunny) [reply state/my_lists/operations/normalize/myLists.js]: Done

File: state/session/operations/__tests__/requestSession.test.js (hunk 4 above)
Comment (pink-bunny) [reply state/session/operations/__tests__/requestSession.test.js]: Done

**Pull request merged**